x/ecocredit/core/fixtures_test.go

```diff
@@ -0,0 +1,17 @@
+package core
+
+import "github.com/cosmos/cosmos-sdk/testutil/testdata"
+
+func genAddress() string {
+	_, _, addr :=
+		testdata.KeyTestPubAddr()
+	return addr.String()
+}
```

Comment:
Should this be added to the `testutil` package? It could be useful beyond ecocredit core.

Reply:
```suggestion
func genAddress() string {
	_, _, addr := testdata.KeyTestPubAddr()
	return addr.String()
}
```